src/components/Login/FooterBar.js

```diff
@@ -25,7 +26,7 @@ const FooterBar = () => {
     }
 
     return (
-        <div className="fixed inset-x-0 bottom-0 bg-white border-t border-gray-400 shadow-md flex justify-around items-center md:hidden z-50">
+        <div className="footer-bar fixed inset-x-0 bottom-0 bg-white border-t border-gray-400 shadow-md flex justify-around items-center md:hidden z-50">
```

Comment:
do we have a class named footer-bar?

Reply:
i created a small class in profile.css file for handling the size in very small screens using a media query so i added the class footer-bar